neurodsp/tests/sim/test_aperiodic.py

```diff
@@ -33,6 +35,26 @@ def test_sim_powerlaw():
     sig = sim_powerlaw(N_SECONDS, FS, f_range=(2, None))
     check_sim_output(sig)
 
+def test_sim_fgn():
+
+    # Simulate white noise.
+    sig = sim_fgn(N_SECONDS, FS)
+    check_sim_output(sig)
+
+    # Check the accuracy of the mean and standard deviation.
+    np.allclose(np.mean(sig), 0, atol=0.1)
+    np.allclose(np.std(sig), 1, atol=0.1)
+
+def test_sim_fbm():
+
+    # Simulate standard brownian motion.
+    sig = sim_fbm(N_SECONDS, FS)
+    check_sim_output(sig)
```

Comment:
to add an accuracy check, we can add a check that the hurst exponent is as expected, using the now merged fluctuations function, right?

Reply:
Perhaps. If such a test were to fail, who would be the culprit? The fluctuations code or the fractional brownian motion code? The accuracy tests I used below these lines were a sanity check on the increments being gaussian by checking the first two moments. In the newest commit I do not normalize the signal and check the first four moments as an accuracy test.